.circleci/config.yml

```diff
@@ -69,21 +80,30 @@ workflows:
     version: 2
     build-test-deploy:
       jobs:
-        - build_and_test:
+        - build_and_test
```

Comment:
i believe this will also require the filter tags to be present or else the release won't run, had something similar happen in the flexo repos